packages/lime-system/files/usr/lib/lua/lime/config.lua

```diff
@@ -26,13 +26,17 @@ function config.sanitize()
 	end
 end
 
-function config.get(sectionname, option)
+function config.get(sectionname, option, default)
 	local limeconf = config.uci:get("lime", sectionname, option)
 	if limeconf then return limeconf end
 
 	local defcnf = config.uci:get("lime-defaults", sectionname, option)
 	if ( defcnf ~= nil ) then
 		config.set(sectionname, option, defcnf)
+	elseif ( default ~= nil ) then
+		defcnf = default
+		local cfn = sectionname.."."..option
+		print("WARNING: Option "..cfn.." is not defined. Using value "..(default or 'False'))
```

Comment:
I think `(default or 'False')` is not needed just appending default should work for any type that may get there (it would work also with nil but we are sure it will not get there in any case because of the elseif)

Reply:
You'r right